Examples/Algorithms/Geant4HepMC/include/ActsExamples/Geant4HepMC/EventRecording.hpp

```diff
@@ -35,6 +38,13 @@ class EventRecording final : public ActsExamples::BareAlgorithm {
     int seed1 = 12345;
     /// random number seed 2
     int seed2 = 45678;
+
+    /// List of processes that can be combined to a single vertex
+    std::vector<std::string> processFilter;
+    /// Process that should be recorded
+    std::string eventSelectionProcess;
+    /// List to veto events with certain processes
+    std::vector<std::string> eventRejectionProcess;
```

Comment:
```suggestion
    /// List of processes that can be combined to a single vertex
    std::vector<std::string> processesCombine;
    /// Process that should be recorded
    std::string processSelect;
    /// List to veto events with certain processes
    std::vector<std::string> processesReject;
```
Suggestion: consistent naming. The `event...` prefix is not needed. It is clear from the algorithm type what is meant here. `event` also has a different meaning here (interaction event) compared to the rest of the code base and this might lead to confusion.

What happens if `processSelect` is empty? From the description and the name I would expect that then nothing gets recorded, but the code suggests otherwise. Could you clarify?

Reply:
Thanks for pointing that out. I changed the description and hope that it clarifies it. Since the class is called "EventRecording" I want to be able to record everything and record certain events only on top of it.
Also changed the naming